CHANGELOG.md

```diff
@@ -101,6 +106,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Support link href
 
 [Unreleased]: https://github.com/cozy/cozy-bar/compare/v4.1.0...HEAD
+[v4.1.1]: https://github.com/cozy/cozy-bar/compare/v4.0.2...v4.1.1
```

Comment:
It should be 
[Unreleased]: https://github.com/cozy/cozy-bar/compare/v4.1.1...HEAD
[v4.1.1]: https://github.com/cozy/cozy-bar/compare/v4.1.0...v4.1.1